### DISCUSSION TITLE
HELP! Model.glb looks too dark on older devices

### DISCUSSION THREAD

**Discussion by Hardin22** [google/model-viewer#4559] [Q&A]
I am encountering an issue where some of my 3D models are rendering significantly darker than expected on specific Android devices using `model-viewer`. This issue seems to be device-specific, as the model renders with appropriate lighting on most devices.

### Issue Description:
- The model appears too dark (not completely black), lacking the standard lighting that is automatically applied and works correctly on other phones.
- This issue is not consistent across all devices only some older lgs and samsungs using chrome , indicating a possible device-specific rendering problem.

### Environment:
- `model-viewer` version: [latest, 3.3.0]
- Affected devices: [unfortunately don't know the exact models, but as today only an old and 1 lg]
- Browser: Chrome 

I have not encountered this issue with other models in the same environment, suggesting it might be specific to this model or how it interacts with `model-viewer`'s lighting on certain devices.
Weirdest thing is that other models i have loaded in the same page seem to work just fine, but all of them  have been processed and made the same exact way. they have equal dimensions.

Could you please provide any insights or suggestions on how to address this issue? Are there known compatibility issues with specific devices or settings that I should be aware of?

Any help or guidance would be greatly appreciated.

**Comment by elalish** [comment]
It's hard to say without knowing what devices or having any screenshots to compare. We do our best to support old devices, but the old ones tend to have a lot more GPU driver bugs that are very hard to work around. I'd guess this is one, so it's unlikely a fix will come.
